payments-core/src/test/java/com/stripe/android/GooglePayJsonFactoryTest.kt

```diff
@@ -18,6 +21,11 @@ class GooglePayJsonFactoryTest {
     private val googlePayConfig = GooglePayConfig(ApiKeyFixtures.DEFAULT_PUBLISHABLE_KEY)
     private val factory = GooglePayJsonFactory(googlePayConfig)
 
+    @AfterTest
+    fun tearDown() {
+        GooglePayJsonFactory.cardBrandFilter = DefaultCardBrandFilter
```

Comment:
I don't think this should be mutable. We should create the factory with the right filter.

Reply:
Adding to my other comment, I think to create the factory with the correct filter we would have to make a series of public API changes in other classes to allow us to inject a CardBrandFilter from PaymentSheet. It's definitely the more correct approach, but not sure I want to expose a CardBrandFilter type just for Google Pay.

https://github.com/stripe/stripe-android/pull/9479